pkg/lakom/config/config_suite_test.go

```diff
@@ -0,0 +1,17 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package config
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestCMD(t *testing.T) {
```

Comment:
I see where this is coming from, but `CMD` is not best name here. Usually the naming scheme is `Test<package-name>`.

 ```suggestion
func TestConfig(t *testing.T) {
```

Reply:
I actually missed renaming this. So I didn't intend for it to be called CMD. Thanks